src/main/java/ua/rent/masters/easystay/service/impl/BookingServiceImpl.java

```diff
@@ -173,4 +182,9 @@ private void validateUpdateDates(BookingRequestUpdateDto requestUpdateDto) {
                     "Check-out date can't be earlier than check-in date");
         }
     }
+
+    private boolean checkUserRole(User user) {
+        return user.getRoles().stream()
+                .anyMatch(role -> role.getName().equals(Role.RoleName.ROLE_MANAGER));
```

Comment:
You are checking if user has manager role, but name of method doesn't look so.
```suggestion
    private boolean isManager(User user) {
        return user.getRoles().stream()
                .anyMatch(role -> role.getName().equals(Role.RoleName.ROLE_MANAGER));
```